pkg/loadbalancer/loadbalancer.go

```diff
@@ -305,6 +305,8 @@ type BackendState uint8
 
 // Backend represents load balancer backend.
 type Backend struct {
+	// FEPortName is the frontend port name. This is used to filter backends sending to EDS.
+	FEPortName string
```

Comment:
Kubernetes mandates using named ports when there are multiple ports in the service spec. How do you account for multiple ports with this variable?

Reply:
Do you have any particular scenario in mind, which it might cause backends got overwritten ?

Based on my understanding, we generate cartensian product for list of services in [here](https://github.com/cilium/cilium/blob/7c86fe77439a328ea905928fb937cb17c3d4e875/pkg/k8s/watchers/watcher.go#L728), then FEPortName is just assigned for look up later.

https://github.com/cilium/cilium/commit/1d5b023b179a066e6f92ea2dbf6699d566cb32bb#diff-ab3c6ac901331db3e778771f91a909772ed5b3c41b0c87e603dede90fd4ad9feR739